samples/mlhelib_tutorial/tut_1_basics.cpp

```diff
@@ -0,0 +1,181 @@
+/*
+* MIT License
+*
+* Copyright (c) 2020 International Business Machines
+*
+* Permission is hereby granted, free of charge, to any person obtaining a copy
+* of this software and associated documentation files (the "Software"), to deal
+* in the Software without restriction, including without limitation the rights
+* to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+* copies of the Software, and to permit persons to whom the Software is
+* furnished to do so, subject to the following conditions:
+*
+* The above copyright notice and this permission notice shall be included in all
+* copies or substantial portions of the Software.
+*
+* THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+* IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+* FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+* AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+* LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+* OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+* SOFTWARE.
+*/
+
+#include <cassert>
+
+#include "helayers/hebase/helib/HelibCkksContext.h"
+#include "helayers/hebase/hebase.h"
+
+// This tutorial shows a basic usage example of using the hebase layer.
+// For our basic example, we'll configure HElib as the underlying HE library
+// and CKKS as the scheme.
+
+using namespace std;
+using namespace helayers;
+
+void tut_1_run(HeContext& he);
+
+void tut_1_basics()
+{
+
+  // First, we initialize the underlying HE library.
+  // We use one of the supplied default configuration.
+  // This performs all the required setup.
+  // NOTE: this also generates a public-private key pair,
+  // so it's ready now for use.
+  HelibCkksContext helib;
+  helib.init(HelibConfig::createCkks8192());
+  // The createCkks8192() method returns a configuration with
+  // Each ciphertext has 8129 slots, i.e., it can hold 8192 numbers.
```

Comment:
maybe somewhere we can mention why slots are important and how to use them?  Its good that we have a default config method that devs can call, but maybe we can help someone understand why to use a specific one?